doc/v2/howto/rnn/index_en.rst

```diff
@@ -1,10 +1,32 @@
 RNN Models
 ==========
+Recurrent neural network(RNN) is an important tool to model sequential data. PaddlePaddle provides flexible interface for building complex recurrent neural network. And we will show how to use PaddlePaddle to build RNN models include the following four main parts.
```

Comment:
Recurrent neural **networks**(RNN) **are** an

Reply:
`And we will show how to use PaddlePaddle to build RNN models include the following four main parts.`
This sentence is grammatically incorrect.

This should be replaced to:

**We will demonstrate how to use PaddlePaddle to build RNN models in the following 4 parts**.